src/elements/Label/Label.js

```diff
@@ -35,31 +34,34 @@ function Label(props) {
     useKeyOnly(circular, 'circular'),
     useKeyOnly(floating, 'floating'),
     useKeyOnly(horizontal, 'horizontal'),
+    useKeyOnly(image, 'image'),
     useKeyOnly(empty, 'empty'),
     useKeyOnly(tag, 'tag'),
     useValueAndKey(attached, 'attached'),
     useKeyOrValueAndKey(corner, 'corner'),
     useKeyOrValueAndKey(pointing, 'pointing'),
+    /* Don't change order of `above` and `below`, they must be after `pointing` */
+    useKeyOnly(above, 'above'),
+    useKeyOnly(below, 'below'),
     useKeyOrValueAndKey(ribbon, 'ribbon'),
```

Comment:
Removed `image` shorthand because image is [prop](https://github.com/Semantic-Org/UI-Label/blob/master/label.css#L56)